src/holofuel/pages/Inbox/Inbox.js

```diff
@@ -369,14 +381,22 @@ export function TransactionRow ({ transaction, setConfirmationModalProperties, i
         showCancelModal={showCancelModal}
         isDeclined={isDeclined}
         isDrawerOpen={isDrawerOpen}
+        areActionsPaused={areActionsPaused}
       />}
     </>}
   </div>
 }
 
-function RevealActionsButton ({ openDrawer, closeDrawer, isDrawerOpen }) {
-  return <div onClick={isDrawerOpen ? closeDrawer : openDrawer} styleName={cx('reveal-actions-button', 'drawer', { 'drawer-close': !isDrawerOpen })} data-testid='reveal-actions-button'>
-    <ForwardIcon styleName='forward-icon' color='#2c405a4d' dataTestId='forward-icon' />
+function RevealActionsButton ({ openDrawer, closeDrawer, isDrawerOpen, areActionsPaused }) {
+  const onClick = isDrawerOpen
+    ? closeDrawer
+    : areActionsPaused
+      ? () => {}
+      : openDrawer
+
+  const iconColor = areActionsPaused ? '#ced5de' : '#2c405a'
+  return <div onClick={onClick} styleName={cx('reveal-actions-button', 'drawer', { 'drawer-close': !isDrawerOpen })} data-testid='reveal-actions-button'>
```

Comment:
Small request: Can we remove the ease-out property on the forward icon whenever `areActionsPaused` is true?  The fact that the opacity change on hover, makes the button feel like it is active/should trigger an action when pressed...

Reply:
Ah, good catch. I was testing on mobile sim so didn't see the hover behavior